recipe/meta.yaml

```diff
@@ -118,141 +77,177 @@ outputs:
-        - {{ cdt('libxau-devel') }}          # [linux]
-        - {{ cdt('libxcb') }}                # [linux]
-        - {{ cdt('libxdamage-devel') }}      # [linux]
-        - {{ cdt('libxext-devel') }}         # [linux]
-        - {{ cdt('libxfixes-devel') }}       # [linux]
-        - {{ cdt('libxi-devel') }}           # [linux]
-        - {{ cdt('libxrender-devel') }}      # [linux]
-        - {{ cdt('libxxf86vm') }}            # [linux]
-        - {{ cdt('mesa-dri-drivers') }}      # [linux]
-        - {{ cdt('mesa-libgl-devel') }}      # [linux]
-        - {{ cdt('xorg-x11-proto-devel') }}  # [linux and build_variant == 'normal']
-      files:
-        - test-cmake/CMakeLists.txt
-        - test-cmake/DisplayImage.cpp
-        - test.cpp
-      commands:
-        - pushd test-cmake
-        - cmake . -GNinja -DCMAKE_BUILD_TYPE=Release
-        - cmake --build . --verbose
-        - popd
-        # Verify dynamic libraries.
-        # "bioinspired", Not working in 3.1.0
-        {% set opencv_libs = [
-             "aruco",
-             "bgsegm",
-             "calib3d",
-             "ccalib",
-             "core",
-             "datasets",
-             "dpm",                               # [linux and x86_64]
-             "face",                              # [linux and x86_64]
-             "features2d",
-             "flann",
-             "fuzzy",
-             "highgui",
-             "imgcodecs",
-             "imgproc",
-             "line_descriptor",
-             "ml",
-             "objdetect",                        # [linux and x86_64]
-             "optflow",
-             "phase_unwrapping",
-             "photo",
-             "plot",
-             "reg",
-             "rgbd",
-             "saliency",
-             "shape",
-             "stereo",
-             "stitching",
-             "structured_light",
-             "superres",
-             "surface_matching",
-             "text",                             # [linux and x86_64]
-             "tracking",
-             "video",
-             "videoio",
-             "videostab",
-             "xfeatures2d",
-             "ximgproc",
-             "xobjdetect",                       # [linux and x86_64]
-             "xphoto",
-        ] %}
-        {% for each_opencv_lib in opencv_libs %}
-        - test -f ${CONDA_PREFIX}/lib/libopencv_{{ each_opencv_lib }}.dylib                        # [osx]
-        - test -f ${CONDA_PREFIX}/lib/libopencv_{{ each_opencv_lib }}.so                           # [linux]
-        - if not exist %CONDA_PREFIX%\\Library\\bin\\opencv_{{ each_opencv_lib }}{{ version|replace(".","") }}.dll exit 1 # [win]
-        {% endfor %}
+        imports:
+          - cv2
+          - cv2.xfeatures2d
+          - cv2.freetype
+        source_files:
+          - test.avi
+        requires:
+          - {{ compiler('c') }}
+          - {{ compiler('cxx') }}
+          - pkg-config                    # [not win]
```

Comment:
We're producing pkg-config files for win, is it possible to easily test this too or would it require a bunch of msys2-related troubleshooting

Reply:
Just need to call pkg-config. But it's limited. You could verify the files visually too.

Reply:
ok